src/test/suite/index.ts

```diff
@@ -5,14 +5,21 @@ import Mocha from "mocha";
 import "source-map-support/register";
 
 export async function run(): Promise<void> {
+    let test_regex = process.env.VSCODE_NEOVIM_TEST_REGEX;
```

Comment:
`git grep -P 'process.env' shows that existing env vars (including test-only env vars) are named like `NEOVIM_xx`.

```suggestion
    let test_regex = process.env.NEOVIM_TEST_REGEX;
```